shepherd_bms/src/compute.cpp

```diff
@@ -282,4 +282,29 @@ void ComputeInterface::sendChargingStatus(bool chargingStatus)
 void ComputeInterface::MCCallback(const CAN_message_t &msg)
 {
     return;
+}
+
+void ComputeInterface::sendCellTemp(uint16_t mCellTemp, uint8_t mCellID, uint16_t minCellTemp, uint8_t minCellID, uint16_t tempAvg)
+{
+    static union
```

Comment:
@dyldonahue I think we gotta rethink making these unions static, I think that the variables shouldn't always be in scope. I was originally thinking static because if you make it a static const the compiler will know to make that constant local to the function but since we aren't using const and don't need to retain its state everytime we don't need static

Reply:
Yeah probably need to do a quick think through to make sure they don't need to be accessed in weird places but off the dome that seems logical